ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/policy-maps-copp.j2

```diff
@@ -0,0 +1,28 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - policy map copp #}
+{% if policy_maps.copp_system_policy.enable is arista.avd.defined %}
+
+### Control-plane Policy Map
+
+#### Control-plane Policy Map Summary
+{%     if policy_maps.copp_system_policy.classes is arista.avd.defined %}
+
+##### copp-system-policy
+
+| Class | Shape | Bandwidth | Rate Unit |
+| ----- | ----- | --------- | --------- |
+{%         for class in policy_maps.copp_system_policy.classes | arista.avd.natural_sort('name') %}
+| {{ class.name }} | {{ class.shape | arista.avd.default("-") }} | {{ class.bandwidth | arista.avd.default("-") }} | {{ class.rate_unit }} |
```

Comment:
if rate_unit is not required pls update here as well

Reply:
command for shape configuration is 'shape pps\kbps <value>'. So we must define rate unit.

Reply:
what if `shape` and `bandwidth` both are not defined.

Reply:
Verified on the lab that class is not getting configured without defining shape and bandwidth.

Reply:
Added default for rate_unit.